modules/virt-must-gather-options.adoc

```diff
@@ -72,7 +74,14 @@ Environment variables are optional. You can run a script by itself or with one o
 
 .Syntax
 
+To collect `must-gather` logs for all operators and products on your cluster in a single pass, run the following command:
+
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --all-images
+----
 
+If you need to pass additional parameters to individual `must-gather` images, use the following explicit syntax:
```

Comment:
@dfitzmau 
This command is not an example, since it has user-replaceable values. How about simply this:
"If you need to pass additional parameters to individual `must-gather` images, use the following command:"